src/Analyzers/CSharp/Analyzers/UsePatternMatching/CSharpUseNotPatternDiagnosticAnalyzer.cs

```diff
@@ -43,7 +43,6 @@ protected override void InitializeWorker(AnalysisContext context)
         {
 #if !CODE_STYLE // CODE_STYLE layer doesn't currently support generating 'not patterns'.  Do not bother analyzing.
```

Comment:
Can we move this outside the protected override so that way the `#if` isn't cutting across the method definitions? It _works_, but it's kinda funky that in one case line 89 is the closing brace of this method and one is the closing brace of SyntaxNodeAction.

Reply:
@sharwell Do we need this `#if` directive now that CodeStyle layer has project references to latest Workspaces and CodeAnalysis projects?

Reply:
`InitializeWorker` is abstract, so we need the definition here sadly. I can clean it up though.

Having said that, the #if !CODE_STYLE could just be placed around the whole file too.

Reply:
> Do we need this #if directive now that CodeStyle layer has project references to latest Workspaces and CodeAnalysis projects?

Seems like no :)